uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/LeaveBalanceChange.php

```diff
@@ -990,9 +987,12 @@ private static function recalculateBalanceChange($balanceChanges, $datesOverlapp
         'expired_balance_change_id' => $balanceChange['id']
```

Comment:
This comment is actually for the line above this one (986). The field name is `expiry_date` and not `expiration_date`. I wonder how this didn't cause any test failures.